src/recipes/tooling-custom-resources.md

```diff
@@ -0,0 +1,40 @@
+<!--
+  ~ Copyright (c) godot-rust; Bromeon and contributors.
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+# Custom Resources
+
+Custom `Resource`s are exposed for the end user to use within their development, but cannot provide any functionality automatically.
+For example, you can create a custom `AudioStream` type that handles a new and interesting audio file type.
+
+
+## Registering a Resource
+
+This workflow is incredibly similar to the [Hello World Example](/intro/hello-world.md)
```

Comment:
```suggestion
This workflow is similar to the [Hello World example](/intro/hello-world.md):
```
No need to be dramatic 😉 I also fixed capitalization and lack of colon.

Also, might be good to explain _why_ `tool` is needed here instead of `// use tool!` as a comment.